name_translation_parameters.rb

```diff
@@ -1,28 +1,51 @@
 require_relative 'rosette_api_error'
 
 class NameTranslationParameters
-  attr_accessor :name, :entity_type, :source_language_of_origin, :source_language_of_use, :source_script,
-                :target_language, :target_scheme, :target_script
+  attr_accessor :entity_type,
```

Comment:
Not sure if this is generally a preference or not, but it's certainly a preference of mine. I prefer to list long, unordered lists vertically and in alphabetical order. I feel it makes them easier to read.